mozilla/fx-private-relay
# Update list of countries everywhere they are listed

This PR addresses MPP-2084 

L10N: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/97 



# New feature description

**Acceptance Criteria**

New content: “Free ⁨Relay⁩ is available in most countries. ⁨Relay Premium⁩ is available in Austria, Belgium, Canada, Cyprus, Estonia, Finland, France, Germany, Greece, Ireland, Italy, Latvia, Lithuania, Luxembourg, Malaysia, Malta, Netherlands, New Zealand, Portugal, Singapore, Slovakia, Slovenia, Spain, Sweden, Switzerland, United Kingdom, and the United States.” 

- update content on homepage FAQ section 
- update content on FAQ page 
- update content on interstitial page (at the top of the page) 

# Screenshot (if applicable)

Not applicable.

# How to test

Make sure content related to where relay is available matches the new content. 

- Visit the homepage FAQ section. 
- Visit the FAQ page
- Visit the Premium page


# Checklist
